src/ResponseMerger/NuxtResponseMerger.php

```diff
@@ -35,6 +35,19 @@ public function mergeResponses(ResponseInterface $backendResponse, ResponseInter
     // global object, this object used to set initial state correctly within
     // the frontend application.
     $init_nuxt_script = file_get_contents(__DIR__ . '/../../assets/nuxt/initNuxt.js');
+
+    // Prepare breadcrumbs string.
+    if (isset($data['breadcrumbs'])) {
+      $init_nuxt_script = str_replace('breadcrumbs: { }', 'breadcrumbs: ' . json_encode($data['breadcrumbs']), $init_nuxt_script);
+      $breadcrumbs = '';
+      foreach ($data['breadcrumbs'] as $crumb) {
+        $url = $crumb['url'];
```

Comment:
I know there are breadcrumb entries which are not linked, so I guess this breaks then.